src/components/SizeIndicator.svelte

```diff
@@ -8,8 +9,10 @@ let match
 let name
 
 $: match = sizes.find(s => s.type === size) || {}
-$: color = match.color || 'secondary'
-$: name = match.name || 'unknown'
+$: color = match.color || '#777'
+$: name = capitalizeFirstLetter(match.name) || 'unknown'
+
+const capitalizeFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1)
```

Comment:
can this be done by css `text-transform: capitalize`?

Reply:
yeah, I had to add a span to get it through to the text node, but worth changing.